code/__DEFINES/reagents.dm

```diff
@@ -184,12 +184,6 @@
 #define GIN 			"gin"
 #define ABSINTHE 			"absinthe"
 #define RUM 			"rum"
-#define TEQUILA 			"tequila"
-#define VERMOUTH 			"vermouth"
-#define WINE 			"wine"
-#define COGNAC 			"cognac"
-#define HOOCH 			"hooch"
-#define ALE 			"ale"
```

Comment:
Captain Pete's Cuban Spiced Rum seems to only contain regular rum. And still, I don't see any recipe to obtain any of the deadrum children.